testsuite/tests/apicast/policy/routing/test_routing_policy_path.py

```diff
@@ -10,19 +10,25 @@
 from testsuite.echoed_request import EchoedRequest
 
 
+@pytest.fixture(scope="module")
+def private_base_url(private_base_url):
+    """Use echo_api as service backend"""
+    return private_base_url("echo_api")
+
+
 @pytest.fixture(scope="module")
 def service_proxy_settings(private_base_url):
     """
     Require compatible backend to be used
     """
-    return rawobj.Proxy(private_base_url("echo_api"))
+    return rawobj.Proxy(private_base_url)
 
 
 @pytest.fixture(scope="module")
 def httpbin_host(private_base_url):
     """Custom hostname to be set via policy"""
 
-    return urlsplit(private_base_url("httpbin")).hostname
+    return urlsplit(private_base_url).hostname
```

Comment:
Why are you changing httpbin to echo api? Why you didn't change name of the fixture?